Client/Clique/Common/Extensions/Extensions.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Extensions.swift
+//  Clique
+//
+//  Created by Infinum on 27.11.2022..
+//
+
+import UIKit
+import SkeletonView
+
+extension UIImageView {
+    
+    func rounded() {
+        layer.borderWidth = 1
+        layer.masksToBounds = false
+        layer.borderColor = UIColor.systemGray5.cgColor
+        layer.cornerRadius = self.frame.height / 2
+        clipsToBounds = true
+    }
```

Comment:
Is this a circle? Use the suggested code. This way you are setting some defaults in the function parameters, meaning you only have to state them when you want to change your defaults, so every time you have some ImageView you can use it just .asCircle(), however if you want bigger border you can just use .asCircle(borderWidth: 5), or color .asCircle(borderColor: .white), also both cases: .asCircle(borderWidth: 5, borderColor: .white) 
```suggestion
    func asCircle(borderWidth: CGFloat = 1, borderColor: UIColor = .systemGray5) {
        layer.borderWidth = borderWidth
        layer.masksToBounds = false
        layer.borderColor = borderColor.cgColor
        layer.cornerRadius = self.frame.height / 2
        clipsToBounds = true
    }
```